handlers/features.go

```diff
@@ -24,6 +24,8 @@ type PostData struct {
 	Examples     []string `json:"examples"`
 	WebhookURL   string   `json:"webhook_url"`
 	FeatureUUID  string   `json:"featureUUID"`
+	AudioLink    string   `json:"audioLink"`
```

Comment:
Also, please can you create a new struct for this:
`type AudioBrieftPostData struct {`
         `"audioLink": "CAPTURE FROM FORM",`
        `"featureUUID": "FROM PREVIOUS PAGE",`
        `"source": "featureBrief",`
        `"examples":[]`
`}`

Reply:
The struct above is used in another endpoint for `userStories`